bundles/org.openhab.binding.hdpowerview/README.md

```diff
@@ -93,20 +93,34 @@ And for horizontal shades, it maps the horizontal position of the "truck" to the
-|--------------------------|-------------------|-----------------------|------------------|----------------|---------|
-| Single action bottom-up  | `position`        | ▲                     | Up               | `OPEN`         | 0%      |
-|                          |                   | ▼                     | Down             | `CLOSED`       | 100%    |
-| Single action top-down   | `position`        | ▲                     | Up               | ***`CLOSED`*** | 0%      |
-|                          |                   | ▼                     | Down             | ***`OPEN`***   | 100%    |
-| Single action right-left | `position`        | ▲                     | ***Left***       | `OPEN`         | 0%      |
-|                          |                   | ▼                     | ***Right***      | `CLOSED`       | 100%    |
-| Single action left-right | `position`        | ▲                     | ***Right***      | `OPEN`         | 0%      |
-|                          |                   | ▼                     | ***Left***       | `CLOSED`       | 100%    |
-| Dual action (lower rail) | `position`        | ▲                     | Up               | `OPEN`         | 0%      |
-|                          |                   | ▼                     | Down             | `CLOSED`       | 100%    |
-| Dual action (upper rail) | ***`secondary`*** | ▲                     | ***Down***       | `OPEN`         | 0%      |
-|                          |                   | ▼                     | ***Up***         | `CLOSED`       | 100%    |
+| Type of Shade            | Channel           | Rollershutter Command | Motion direction | Shade State    | Percent | Pebble Remote Button         |
+|--------------------------|-------------------|-----------------------|------------------|----------------|---------|------------------------------|
+| Single action<br>bottom-up  | `position`        | ▲                     | Up               | `OPEN`         | 0%      | ▲                            |
+|                             |                   | ▼                     | Down             | `CLOSED`       | 100%    | ▼                            |
+| Single action<br>top-down   | `position`        | ▲                     | Up               | ***`CLOSED`*** | 0%      | ▲                            |
+|                             |                   | ▼                     | Down             | ***`OPEN`***   | 100%    | ▼                            |
+| Single action<br>right-left | `position`        | ▲                     | ***Left***       | `OPEN`         | 0%      | ▲                            |
+|                             |                   | ▼                     | ***Right***      | `CLOSED`       | 100%    | ▼                            |
+| Single action<br>eft-right | `position`        | ▲                     | ***Right***      | `OPEN`         | 0%      | ▲                            |
```

Comment:
The updated ReadMe looks great! Just found one very small typo: missing l (L) in "left", after the <br>.